.pre-commit-config.yaml

```diff
@@ -0,0 +1,18 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.0.1
+    hooks:
+      - id: check-docstring-first
+      - id: check-toml
+      - id: check-yaml
+        exclude: packaging/.*
+      - id: end-of-file-fixer
+
+  - repo: https://github.com/omnilib/ufmt
+    rev: v1.3.0
+    hooks:
+      - id: ufmt
+        args: [check]
```

Comment:
Seems like there is an issue with this line.

https://app.circleci.com/pipelines/github/pytorch/audio/8662/workflows/b11596e4-bc97-4954-92d9-336393795601/jobs/460162